ci/steps/publish-npm.sh

```diff
@@ -51,6 +51,27 @@ main() {
     exit 1
   fi
 
+  # Check that we're using at least v7 of npm CLI
+  if ! command -v npm &> /dev/null; then
+    echo "Couldn't find the npm CLI"
+    echo "Are you sure you have it installed?"
+    echo "Please check again with: npm -v"
+    echo "And re-run the script."
+    exit 1
+  fi
+
+  # NOTE@jsjoeio - this needs to run inside the release dir
+  # where the package.json for code-server is.
+  pushd release
+  if npm pkg get name && [ $? -eq 1 ]; then
```

Comment:
@code-asher how's this?

Reply:
I think this ends up using the exit code of `[` so it never enters the `if`.

I tested with:

```
bash -c 'if npm pkg get name && [ $? -eq 1 ]; then echo hello ; fi'
```

This should work:

```
if ! npm pkg get name ; then
```

We might also want to redirect the output because `npm` outputs a whole bunch of junk:

```
if ! npm pkg get name > /dev/null ; then
```

Reply:
I know it no longer matters because we are using `jq` but there must be something else at play.  I tried changing `-eq 1` to `eq 0` and it still does not enter the `if` block.  What is bash doing to me?

Reply:
ohhhhhh right because `npm pkg get name` exits with 1 so it aborts the `&&`

Reply:
ohhh that's really strange 🤔 LOL well makes me glad we didn't go with this solution because I probably would have introduced a bug. I wonder if @deansheather knows

Reply:
Ahhh  that makes sense!

Reply:
Er, I knew that 😉

Reply:
ahahahahaha just keeping my sharp, I appreciate it!  :joy: